src/ngraph/runtime/cpu/CMakeLists.txt

```diff
@@ -157,5 +149,12 @@ if (NGRAPH_CPU_ENABLE)
     target_include_directories(cpu_backend SYSTEM PUBLIC libmkldnn)
     set_target_properties(cpu_backend PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${NGRAPH_BUILD_DIR})
 
+    if(NGRAPH_DISTRIBUTED_ENABLE)
+        add_definitions(-DNGRAPH_DISTRIBUTED)
+        add_dependencies(cpu_backend libmlsl)
+        target_include_directories(cpu_backend SYSTEM PUBLIC libmlsl)
```

Comment:
Can we make both the include_directories and link_libraries PRIVATE? The less we expose the better.

Reply:
I've changed PUBLIC to PRIVATE, thanks for the comment